### PR TITLE
allow for module use in goplay

### PR DESCRIPTION
**Pull request by schultzor**
👋 no idea if you still use this, but I came across a year or so ago and finally got around to making it work with go modules

### CODE REVIEW

File: cmd/goplay/main.go [hunk 1]
```diff
@@ -34,10 +34,13 @@ func main() {
 }
 `
 
+var cwd string
```
Comment (fhs) [inline cmd/goplay/main.go]: Don't use a global. Pass `cwd` as an extra argument.

Actually, you don't need this if you just `os.Chdir` in `main` function.

File: cmd/goplay/main.go [hunk 2]
```diff
@@ -51,11 +54,18 @@ func main() {
...
 		log.Fatal(err)
 	}
+	cwd = dir
+
 	defer os.RemoveAll(dir)
 	file := path.Join(dir, "a.go")
 	if err := ioutil.WriteFile(file, []byte(HelloProg), 0600); err != nil {
 		log.Fatal(err)
 	}
+	modInitCmd := exec.Command("go", "mod", "init", "foo.bar/goplay")
```
Comment (fhs) [inline cmd/goplay/main.go:63]:
```suggestion
	modInitCmd := exec.Command("go", "mod", "init", "example.com/goplay")
```

File: cmd/goplay/main.go [hunk 3]
```diff
@@ -51,11 +54,18 @@ func main() {
...
+	modInitCmd.Dir = dir
```
Comment (fhs) [inline cmd/goplay/main.go]: Just call `os.Chdir` so you don't need this.

**Review by fhs — CHANGES_REQUESTED** [review]
Nice! I don't use it as often as I should.

File: cmd/goplay/main.go (hunk 3 above)
Comment (schultzor) [reply cmd/goplay/main.go]: 👍

**Review by fhs — APPROVED** [review]
:+1:

**Pull request merged**